### PR TITLE
DOCAPI-7415: Docs for the -Resample aggregate function combinator.

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation for #5590.

### CODE REVIEW

**Comment by BayoNet** [comment]
@blinkov Do you have any comments?

File: docs/en/query_language/agg_functions/combinators.md [hunk 1]
```diff
@@ -44,5 +44,71 @@ Merges the intermediate aggregation states in the same way as the -Merge combina
 
 Converts an aggregate function for tables into an aggregate function for arrays that aggregates the corresponding array items and returns an array of results. For example, `sumForEach` for the arrays `[1, 2]`, `[3, 4, 5]`and`[6, 7]`returns the result `[10, 13, 5]` after adding together the corresponding array items.
 
+## -Resample
+
+Allows to aggregate data, separating them into intervals.
```
Comment (blinkov) [inline docs/en/query_language/agg_functions/combinators.md]: this sentence sounds confusing and hardly makes it possible to understand the feature without reading further, let's make it more detailed?

File: docs/en/query_language/agg_functions/combinators.md [hunk 2]
```diff
@@ -44,5 +44,71 @@ Merges the intermediate aggregation states in the same way as the -Merge combina
...
+
+```
+<aggFunction>Resample(start, end, step)(<aggFunction_params>, resampling_key)
+```
+
+**Parameters**
+
+- `start` — Starting value of the whole required interval for the values of `resampling_key`.
```
Comment (blinkov) [inline docs/en/query_language/agg_functions/combinators.md]: * Starting/ending or start/end
* We need to mention whether those borders are inclusive or not

File: docs/en/query_language/agg_functions/combinators.md [hunk 3]
```diff
@@ -44,5 +44,71 @@ Merges the intermediate aggregation states in the same way as the -Merge combina
...
+- `stop` — End value of the whole required interval for the values of `resampling_key`.
+- `step` — Step for separating the whole interval by subintervals. The `aggFunction` is executed over these subintervals.
```
Comment (blinkov) [inline docs/en/query_language/agg_functions/combinators.md]: ...over each of those subintervals independently?

File: docs/en/query_language/agg_functions/combinators.md [hunk 4]
```diff
@@ -44,5 +44,71 @@ Merges the intermediate aggregation states in the same way as the -Merge combina
...
+```sql
+SELECT groupArrayResample(30, 75, 30)(name, age) from people
+```
+```text
+┌─groupArrayResample(30, 75, 30)(name, age)─────┐
+│ [['Alice','Mary','Evelyn'],['David','Brian']] │
+└───────────────────────────────────────────────┘
+```
+
+What do we see in the result?
```
Comment (blinkov) [inline docs/en/query_language/agg_functions/combinators.md]: do we use this question style in docs?

File: docs/en/query_language/agg_functions/combinators.md [hunk 5]
```diff
@@ -44,5 +44,71 @@ Merges the intermediate aggregation states in the same way as the -Merge combina
...
+
+`Jonh` is out of the sample because he is too young. Other persons are distributed according to the specified age intervals.
```
Comment (blinkov) [inline docs/en/query_language/agg_functions/combinators.md]: persons -> people?

**Review by blinkov — CHANGES_REQUESTED** [review]

**Pull request merged**